Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -331,17 +331,21 @@ void ABenchmarkGymGameModeBase::TickPlayersConnectedCheck(float DeltaSeconds)
 
 	if (RequiredPlayerCheckTimer.HasTimerGoneOff() && !DeploymentValidTimer.HasTimerGoneOff())
 	{
-		if (SmoothedTotalAuthPlayers < RequiredPlayers)
+		const int32* ActorCount = TotalActorCounts.Find(SimulatedPawnClass);
+		if (ActorCount != nullptr)
 		{
-			bHasRequiredPlayersCheckFailed = true;
-			// This log is used by the NFR pipeline to indicate if a client failed to connect
-			NFR_LOG(LogBenchmarkGymGameModeBase, Error, TEXT("%s: Client connection dropped. Required %d, got %.1f"), *NFRFailureString, RequiredPlayers, SmoothedTotalAuthPlayers);
-		}
-		else
-		{
-			RequiredPlayerCheckTimer.SetTimer(10);
-			// Useful for NFR log inspection
-			NFR_LOG(LogBenchmarkGymGameModeBase, Log, TEXT("All clients successfully connected. Required %d, got %.1f"), RequiredPlayers, SmoothedTotalAuthPlayers);
+			if ((*ActorCount) < RequiredPlayers)
```

Comment:
Nit + opinion: Don't feel brackets are nessesary
```suggestion
			if (*ActorCount < RequiredPlayers)
```